src/contentTypeConfig.js

```diff
@@ -255,7 +255,31 @@ let contentTypeConfig = {
             `"description": ${descriptionField}`,
             'total_xp',
         ]
-    }
+    },
+    'foundation': {
+        'fields': [
+            `"description": ${descriptionField}`,
+            `"instructors":instructor[]->name`,
+            `published_on`,
+            `"id": railcontent_id`,
+            `title`,
+            `"type": _type`,
```

Comment:
These fields are included in the DEFAULT_FIELDS already. 
It looks like sanity handles merges the duplicated fields, but 
```suggestion
```